spec/spec_helper.rb

```diff
@@ -6,8 +6,10 @@
 require 'rspec'
 require 'database_cleaner'
 require 'sequel'
+require 'rack/test'
 
 require_relative '../app/models/shop'
+require_relative '../myapp.rb'
```

Comment:
Now that you are requiring `myapp.rb` in your spec_helper, you no longer need line 11. Because `myapp.rb` already requires `app/models/shop`